apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -438,6 +438,12 @@ const (
 	// - "False" means some of them have failed.
 	// - "Unknown" means we haven't started the apply yet.
 	ResourcesAppliedConditionType ResourcePlacementConditionType = "ResourceApplied"
+
+	// ResourceScheduledConditionType indicates whether we have successfully scheduled the selected resources.
+	// Its condition status can be one of the following:
+	// - "True" means we have successfully scheduled the resources to satisfy the placement requirement.
+	// - "False" means we didn't fully satisfy the placement requirement. We will fill the Message field.
+	ResourceScheduledConditionType ResourcePlacementConditionType = "ResourceScheduled"
```

Comment:
it's not possible to be "unknown". we will only populate the ResourcePlacementStatus only when scheduler finishes the scheduling and the result is either "succeed" or "fail".

> I assume that this is the condition to indicate if the we have updated the binding to the latest resource snapshot, right?
> 
> In this case, maybe the description needs a bit more clarification.
i'm going to add another new condition type to reflect that. It will be in a separate PR when collecting work status